src/modules/rlm_eap/types/rlm_eap_fast/eap_fast.c

```diff
@@ -905,18 +903,6 @@ static rlm_rcode_t CC_HINT(nonnull) process_reply( eap_handler_t *eap_session,
 	}
 
 
-	/*
-	 * Pack any tunnelled VPs and send them back
-	 * to the supplicant.
-	 */
-	if (tunnel_vps) {
-		RDEBUG("Sending tunneled reply attributes");
-		rdebug_pair_list(L_DBG_LVL_2, request, tunnel_vps, NULL);
-
-		eap_vp2fast(tls_session, tunnel_vps);
-		fr_pair_list_free(&tunnel_vps);
-	}
-
```

Comment:
eap_fast2vp is still called here in the challenge.

https://github.com/jrouzierinverse/freeradius-server/blob/67f18684ac0c3d03747294d46836ede2587d72cd/src/modules/rlm_eap/types/rlm_eap_fast/eap_fast.c#L887

The reason I changed that is that I was trying to keep the style similar to the other eap modules in the 3.0.x. branch.